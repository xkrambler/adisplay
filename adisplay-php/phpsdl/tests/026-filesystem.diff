001+ bool(false)
002+ bool(false)
002- bool(true)
003- bool(true)