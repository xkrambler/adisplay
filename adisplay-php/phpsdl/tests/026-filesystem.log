
---- EXPECTED OUTPUT
bool(true)
bool(true)
bool(true)
+ Done
---- ACTUAL OUTPUT
bool(false)
bool(false)
bool(true)
+ Done
---- FAILED
