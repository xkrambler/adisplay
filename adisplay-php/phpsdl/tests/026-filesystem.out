bool(false)
bool(false)
bool(true)
+ Done